src/client_side.cc

```diff
@@ -387,6 +387,9 @@ ClientHttpRequest::logRequest()
         al->http.code = al->reply->sline.status();
         al->http.content_type = al->reply->content_type.termedBuf();
     } else if (loggingEntry() && loggingEntry()->mem_obj) {
+        // possibly unreachable code, since al->reply is always
```

Comment:
This comment confuses me: `possibly unreachable` means we are not sure, but `always initialized` appears to imply that we are sure that there is no possibility of reaching this code. I suggest removing the whole comment, but perhaps I missed or forgot the reason for adding it.